tests/tests.js

```diff
@@ -108,13 +108,17 @@ module.exports = {
     },
     'Test width and height Properties and null and empty strings': function (test) {
         var style = new cssstyle.CSSStyleDeclaration();
-        test.expect(7);
+        test.expect(9);
         style.height = 6;
         test.ok('' === style.height, 'height does not remain unset');
         style.width = 0;
         test.ok('0px' === style.width, 'width is not 0px');
         style.height = '34%';
         test.ok('34%' === style.height, 'height is not 34%');
+        style.height = '100vh';
+        test.ok('100vh' === style.height, 'height is not 100vh');
+        style.height = '100vw';
+        test.ok('100vw' === style.height, 'height is not 100vw');
```

Comment:
These assertion names are misleading. We are testing that the height **is** 100vh / 100vw. So it should read `height is 100vw` or `height: 100vw` ... can you update the two new tests and the two above it, please?